spritz/app/src/main/res/values/styles.xml

```diff
@@ -16,4 +16,9 @@
 
     <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
 
+    <style name="LoveText">
```

Comment:
This seems unused, shouldn't it be assigned to [the textview](https://github.com/novoda/android-demos/pull/37/files#diff-bf21b1e2396e27b6b06fbd91485263bfR45)?